app/src/getters/getChainConfig.ts

```diff
@@ -0,0 +1,5 @@
+import * as config from '@streamr/config'
+export const defaultChainConfig: config.Chain =
+    process.env.HUB_CONFIG_ENV === 'production'
```

Comment:
`HUB_CONFIG_ENV` was added only to point to an alternative .toml file. If anything, using `NODE_ENV` should be used.. somehow.

Also, do we use a fallback? This env var is not always set, and for production envs it's definitely NOT set.

@pmichalowski thoughts?

Reply:
Actually we don't have a fallback so I'll make it. As for the HUB_CONFIG_ENV the issue was that for some reason if you set the NODE_ENV to production, the local dev server won't run - it will throw some Sentry errors so we switched to use a different env variable to change config. I'd stick with HUB_CONFIG_ENV now and make a ticket to solve the issue with the local dev server startup with NODE_ENV=production